src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -53,6 +53,7 @@ enum Format {
     Protobuf {
         descriptor_file: String,
         message: String,
+        confluent_wire_format: bool,
```

Comment:
could we add an optional `schema_message_index: u8` field here (defaulting to `0`) so that we can test the error behavior? (and then also test the error behavior.)